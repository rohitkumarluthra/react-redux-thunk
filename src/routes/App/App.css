.row {  
    display: flex;
    flex-wrap: wrap;
}

.main {
  flex: 80%;
  /* background-color: #f1f1f1; */
  padding: 20px;
  height: 100vh;
}