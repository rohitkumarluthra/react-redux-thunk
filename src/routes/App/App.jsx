import React from 'react';

import "./App.css";
import SideNav from '../SideNav/SideNav';
import Main from '../Main/Main';

const App = props => (
  <div className="row">
    <SideNav />
    <div className="main">
      <Main/>
    </div>
  </div>
);

export default App;