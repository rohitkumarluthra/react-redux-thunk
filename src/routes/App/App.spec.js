import React from 'react';
import { act } from 'react-dom/test-utils';
import App from './App';
import '../../setupTests';
import renderer from 'react-test-renderer';
import store from '../../redux/store';
import { Provider } from "react-redux";

describe("App Component", () => {
  it("should render without throwing an error", () => {
    let rendered;

    act(() => {
      rendered = renderer.create(
        <Provider store={store}>
            <App />
        </Provider>
      )
      expect(rendered.toJSON()).toMatchSnapshot();
    });
    
  });
});