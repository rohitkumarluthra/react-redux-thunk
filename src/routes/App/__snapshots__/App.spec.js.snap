// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component should render without throwing an error 1`] = `
<div
  className="row"
>
  <div
    className="side"
  >
    <h2>
      Categories
    </h2>
    <div
      className="navbar"
    >
      <ul />
    </div>
  </div>
  <div
    className="main"
  >
    <div
      className="main"
    >
      <div
        className="wrapper"
      >
        <div
          className="spinner"
          style={
            Object {
              "display": "block",
              "fontSize": "3em",
              "left": "60%",
              "top": "60%",
            }
          }
        >
          <span
            className="fa fa-circle-o-notch fa-spin"
          />
        </div>
      </div>
      <div
        className="btn_wrapper"
      >
        <button
          className="btn_more"
          onClick={[Function]}
        >
          More
        </button>
      </div>
    </div>
  </div>
</div>
`;
